src/pages/Login/SignUp.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import SignInput from '../../components/Sign/SignInput';
+import SignBtn from '../../components/Sign/SignBtn';
+import inputData_list from '../../components/Sign/InputData';
+import './signup.scss';
+
+class SignUp extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      userInfo: {},
+    };
+  }
+
+  getValue = e => {
+    const { name, value } = e.target;
+    this.setState({
+      userInfo: { ...this.state.userInfo, [name]: value },
+    });
+  };
+
+  // 비밀번호 확인 유효성 검사 기능 구현
+  pwValidationCheck = e => {
+    const { value } = e.target;
+    let password = this.state.userInfo.password;
```

Comment:
- const 로 선언해주시고,
- 구조분해 할당 문법을 이용해 주세요.
```suggestion
    const { password } = this.state.userInfo;
```

Reply:
확인! 수정완료했습니다 :)